src/main/resources/static/javascript/mainApp/admin.controller.js

```diff
@@ -26,23 +26,31 @@
         $scope.selectedGroupingsPaths = [];
         $scope.emptySelect = false;
         $scope.selectedOwnedGroupingsNames = [];
+        $scope.groupingsLoading = false;
+        $scope.userGroupingInformationLoading = false;
```

Comment:
I'm not completely sure which variable controls which spinner. Could you make these variables more specific? Thanks.

- [ ] Rename variables to be more specific